ksql-execution/src/test/java/io/confluent/ksql/execution/util/ExpressionTypeManagerTest.java

```diff
@@ -444,7 +448,13 @@ public void shouldFailOnInconsistentWhenResultType() {
         Optional.empty()
     );
     expectedException.expect(KsqlException.class);
-    expectedException.expectMessage("Invalid Case expression. Schemas for 'THEN' clauses should be the same. Result schema: Schema{STRING}. Schema for THEN expression 'WHEN (TEST1.COL0 = 10) THEN 10' is Schema{INT32}");
+    expectedException.expectMessage(
+        "Invalid Case expression. Schemas for all 'THEN' clauses should be the same."
```

Comment:
Might be more user friendly to say 'Types for all 'THEN' clauses...' (Schema might be a bit confusing to the user in this context)